src/encoder/encodervorbis.cpp

```diff
@@ -37,39 +37,15 @@ A lot of stuff has been stolen from:
 
 EncoderVorbis::EncoderVorbis(EncoderCallback* pCallback)
         : m_bStreamInitialized(false),
+          m_vblock({}),
```

Comment:
Initializing m_vblock is redundant. It is proper initialized by vorbis_block_init(). 
Using gcc 4.8 we remove the code checker warning by introducing a compiler warning. 
I would prefer to silence the code checker here, since it is a actually a false positive.